tests/sandbox/create_message_batches/test_retry_too_early.py

```diff
@@ -0,0 +1,95 @@
+import requests
+import pytest
+import uuid
+from lib import Assertions, Generators
+from lib.constants import CORRELATION_IDS
+
+FORBIDDEN_TOKEN = {"Authorization": "Bearer ClientNotRecognised"}
+TRIGGER_425_ROUTING_CONFIG_ID = "d895ade5-0029-4fc3-9fb5-86e1e5370854"
+
+
+@pytest.mark.sandboxtest
+@pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
+def test_425_retry_too_early(nhsd_apim_proxy_url, correlation_id):
+    """
+    .. py:function:: An API consumer retrying a request while the \
+        original request is still being processed receives a 425 error
+
+        | **Given** the API consumer retries a request
+        | **When** the original request is still being processed
+        | **Then** the response returns a 425 retry too early error
+
+    **Asserts**
+    - Response returns a 425 'Retry Too Early' error
+    - Response returns the expected error message body referencing the Authorization header
```

Comment:
need to remove the `referencing the Authorization header` - I'm wondering if the message should reference the /data/attributes/messageBatchReference field as that is the unique identifier for the batch and what we key the idempotency check off?

Reply:
Is there a chance in the future that we will have 425s returned from other endpoints with idempotency based off a different field?

Reply:
Yes, as per our discussion earlier it would be really nice to have this source value being different on a per endpoint basis